auto_nag/constants.py

```diff
@@ -3,7 +3,7 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-HIGH_PRIORITY = {"P1", "P2"}
+HIGH_PRIORITY = {"P1"}
 LOW_PRIORITY = {"P3", "P4", "P5"}
```

Comment:
```suggestion
LOW_PRIORITY = {"P2", "P3", "P4", "P5"}
```

Reply:
`P2` is still a high priority, but we want to ignore it in some cases to reduce noise for the reason that you mentioned above. Thus, it could be less confusing to do that explicitly.

Example, change the following to be `bug["priority"] != "P1"`:
https://github.com/mozilla/relman-auto-nag/blob/9e4aa8375f7c74e761fbddb9c62f6b565d542b57/auto_nag/scripts/assignee_no_login.py#L102

Reply:
The priorities are not well defined, so any priority level could be considered either high or low. We can only be sure P1 is high and P3 and below are low, but we can't say much about P2 (some teams consider it as "next after P1", some others consider it as a kind of backlog but more important than P3).

We could do it explicitly, but it would mean removing "HIGH_PRIORITY" completely (as we'd remove it in assignee_no_login and inactive_ni_pending, which are the only two users of the constant).
I'd keep it so that we can more easily update it in a single place if in the future we have better definitions for priority, but I'm OK if you prefer to have explicit P1 in the scripts and remove the usage of "HIGH_PRIORITY".

Reply:
I see your point. We could add a comment so it will be clear that missing `P2` is intentional, or instead we could add `MEDIUM_PRIORITY = {"P2"}` (without using it).

Reply:
I like the idea of adding MEDIUM_PRIORITY, done in f9caf5e.